pylint_odoo/checkers/no_modules.py

```diff
@@ -371,11 +377,21 @@ def colon_list_to_dict(self, colon_list):
         """
         return dict([item.split(":") for item in colon_list])
 
+    def replace_and_first_lower(self, string):
+        """ Conver 'Hello World' to 'hello_World'
```

Comment:
I try to delete this initial space but the pep8 fail! 

![initial space pep8](https://cloud.githubusercontent.com/assets/1387970/21522935/1e627d62-cce0-11e6-9892-7250ef7b2c8a.png)